development-process.md

```diff
@@ -0,0 +1,144 @@
+# ODL Development Process
+An approximate workflow guide to hacking on software at the MIT Office of Digital Learning.
+
+# Purpose
+Every team of developers has its own particular approach to writing, reviewing, and deploying code.
+This guide aims to capture MIT ODL's way of doing things. It will be (hopefully) useful as a
+step-by-step guide for new developers.
+
+# Tools
+
+- git
+- Zenhub Chrome extension (https://www.zenhub.io/)
+  - This extension adds the **Boards** and **Burndown** tabs to Github repos. We use it in the way
+  that you might use Jira or Trello. The **Boards** tab is particularly important as it adds a
```

Comment:
JIRA

and, fwiw, we have tried Trello. At the time the github integration was too weak. Not sure if it's gotten better.